src/model.meta

```diff
@@ -14,6 +14,12 @@
   type = integer
   units = count
   dimensions = ()
+[ aerosol_model_name ]
+  standard_name = aerosol_model_name
+  units = 1
+  type = character
+  kind = len=*
+  dimensions = ()
```

Comment:
Moving this to a module takes it out of the call list (not needed except at init time).